### PR TITLE
i18n: Mark strings for translation

### PR DESCRIPTION
**Pull request by tombettany**
In preparation for i18n, mark all the appropriate strings for
translation.

### CODE REVIEW

File: kano_login/login.py [hunk 1]
```diff
@@ -233,8 +234,8 @@ def log_in_success(self):
             cmd = '{bin_dir}/kano-sync --sync -s'.format(bin_dir=bin_dir)
             run_bg(cmd)
 
-        title = "Success!"
-        description = "You're in - online features now enabled."
+        title = _("Success!")
+        description = _("You're in - online features now enabled.")
         return_value = "SUCCESS"
```
Comment (alex5imon) [inline kano_login/login.py]: does it need to be translated?

File: kano_login/login.py [hunk 2]
```diff
@@ -189,7 +190,7 @@ def log_user_in_with_internet(self):
 
         if not success:
             logger.info('problem with login: {}'.format(text))
-            title = "Houston, we have a problem"
+            title = _("Houston, we have a problem")
             description = text
             return_value = "FAIL"
```
Comment (alex5imon) [inline kano_login/login.py]: needs translation?

File: kano_registration_gui/GetData.py [hunk 3]
```diff
@@ -215,7 +215,9 @@ def __init__(self, age):
         self.t_and_cs.connect("t-and-cs-clicked",
                               self.show_terms_and_conditions)
 
-        self.marketing_checkbutton = Gtk.CheckButton.new_with_label("Email me cool stuff to\ndo with my Kano")
+        self.marketing_checkbutton = Gtk.CheckButton.new_with_label(
+            _("Email me cool stuff to\ndo with my Kano")
```
Comment (alex5imon) [inline kano_registration_gui/GetData.py]: Does this work well with the `\n`?

**Comment by alex5imon** [comment]
Seems okay to me

File: kano_registration_gui/GetData.py (hunk 3 above)
Comment (tombettany) [inline kano_registration_gui/GetData.py]: Good question, you are right I think that we should do something like:
```python
self.marketing_checkbutton = Gtk.CheckButton.new_with_label(
    _("Email me cool stuff to do with my Kano")
chk_btn_label = self.marketing_checkbutton.get_child()
chk_btn_label.set_max_width_chars(20)
chk_btn_label.set_line_wrap(True)
```

**Comment by tombettany** [comment]
@alex5imon I fixed the problem with newline characters in strings marked for translation. Now merging.

**Pull request merged**